gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jun 21 03:16:17 IST 2018
+#Fri Dec 21 03:04:02 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
You should not downgrade the Gradle version.

Reply:
Okay sir.I will take care next time.